src/javascript/app/pages/cashier/cashier.js

```diff
@@ -113,7 +114,7 @@ const Cashier = (() => {
         const currency_hint = Currency.isCryptocurrency(currency)
             ? localize('Don\'t want to trade in [_1]? You can open another cryptocurrency account.', `${Currency.getCurrencyDisplayCode(currency)}`) + account_action_text
             : has_no_mt5 && has_no_transaction
-                ? localize('You can [_1]set a new currency[_2] before you deposit for the first time or create an MT5 account.', can_change ? [`<a href=${Url.urlFor('user/accounts')}>`, '</a>'] : ['', ''])
+                ? localize('You can [_1]set a new currency[_2] before you deposit for the first [_3].', can_change ? [`<a href=${Url.urlFor('user/accounts')}>`, '</a>', `${clients_country === 'im' ? 'time' : 'time or create an MT5 account'}`] : ['', '', ''])
```

Comment:
I might be mistaken, but won't this condition print a string without the last word: 'You can set a new currency before you deposit for the first .' in case if `can_change` is false?

Reply:
Indeed, you are correct, I fixed this now by splitting up this ternary so that I can add my change to it without being affected.